HW_6/task_1.py

```diff
@@ -0,0 +1,13 @@
+"""
+Напишите программу вычисления арифметического выражения заданного строкой.
+Используйте операции +,-,/,*. приоритет операций стандартный.
+По возможности реализуйте использования скобок, меняющих приоритет операций.
+
+Ввод: значение типа <str>
+Вывод: значение числового типа данных
+"""
+
+test_str =str("45 + 10 * 2 - (6 + 9 / 3)")
```

Comment:
```suggestion
test_str = "45 + 10 * 2 - (6 + 9 / 3)"
```
Это ведь и так строка